src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -118,73 +118,73 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AssemblyNameDoesNotExist" xml:space="preserve">
-    <value>Assembly with name '{0}' does not exist at {1}.</value>
+    <value>'{0}' assembly does not exist at {1}.</value>
   </data>
   <data name="AssemblyNameDoesNotMatch" xml:space="preserve">
-    <value>{2} assembly name '{0}' does not match with {3} assembly name '{1}'.</value>
+    <value>'{0}' assembly name does not match '{1}'.</value>
   </data>
   <data name="AssemblyPublicKeyTokenDoesNotMatch" xml:space="preserve">
-    <value>{2} assembly public key token '{0}' does not match with {3} '{1}'.</value>
+    <value>'{0}' assembly public key token does not match '{1}'.</value>
   </data>
   <data name="AssemblySearchDirectoryDoesNotExist" xml:space="preserve">
-    <value>Provided assembly search directory '{0}' does not exist.</value>
+    <value>Provided search directory '{0}' does not exist.</value>
   </data>
   <data name="CannotAddAbstractMember" xml:space="preserve">
-    <value>Cannot add abstract member '{0}' to {1} because it does not exist on {2}</value>
+    <value>Cannot add abstract member '{0}' to {1}.</value>
```

Comment:
```suggestion
    <value>Cannot add abstract member '{0}' to '{1}'.</value>
```
is {1} a string? If so it should be quoted too. We should always quote strings, never quote numbers.

Reply:
{0} is a string.
{1} and {2} are `MetadataInformation`